src/gpg/verify.py

```diff
@@ -20,17 +21,22 @@ def verify(signature_file, filename):
         print(e)
 
     if verified is True:
-        for i in range(len(result.signatures)):
-            sign = result.signatures[i]
-            print(  
-                'Signature made {2}\n'
-                '\tusing key {1}\n'
-                'Good signature from "{0}"'
-                .format(c.get_key(sign.fpr).uids[0].uid,
-                sign.fpr, time.ctime(sign.timestamp))         
-            )
+        for signature in result.signatures:
+            user = c.get_key(signature.fpr).uids[0].uid
+            fpr = signature.fpr
+            signed_time = time.ctime(signature.timestamp)
+
+            message = '''
+                      Good signature from {user}
+                      with key {fingerprint}
+                      made at {time}
+                      '''.format(user=user,
+                                 fingerprint=fpr,
+                                 time=signed_time)
+
+            print(textwrap.dedent(message))
```

Comment:
dedent was required as inside triple quotes, when we change the line, the spaces due to indentation are also printed. One way to remove them is to move the string forward something like
```
            signed_time = time.ctime(signature.timestamp)

            message = '''
Good signature from {user}
with key {fingerprint}
made at {time}
                      '''.format(user=user,
                                 fingerprint=fpr,
                                 time=signed_time)

            print(textwrap.dedent(message))
```
which looks a bit ugly

Reply:
sample output
```
Good signature from test <test@test.org>
using key F5524BE5F6BC3664A62DD7E49CC7AD0CDC0AC5CE
made at Sun Jun  3 23:15:22 2018
```